tests/ro/creator/data/katoomba-rainfall/example_blueprint.yml

```
@@ -0,0 +1,59 @@
+  content-type: text/csv; charset=utf8
+
+file:///specialfile: # additional individual file
+  fileSize: 1MB
+  content-type: application/octet-stream
+
+file:///first.csv: # correcting autofill for this case
+  name: My very firstly named csv
+  content-type: text/csv; charset=latin-1
+
+https://example.org/whatever.csv: # external content
+  autofill: off
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)